frontend/tsconfig.json

```diff
@@ -1,6 +1,22 @@
 {
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
-    "strict": true
-  }
+    "strict": true,
+    "baseUrl": ".",
+    "paths": {
```

Comment:
Good addition! You might eventually need to add more aliases. For example, `@/screen`, so make sure your teammates know how to use import alias.